### ISSUE TITLE
feature_request(lint): disable for blockquotes and quotes

### ISSUE THREAD

**Issue by Kristinita** [zerok/remark-lint-write-good#5]
### 1. Summary

It would be nice, if remark-lint-write-good will not lint blockquotes and text between “quotes” (and «quotes» for another languages).

### 2. Example

+ `Kira_remark-lint-write-good__blockquotes_and_qoutes--non-valid-example.md`

```md
# Blockquote

> And therefore must his choice be circumscribed

# Quotes

“Getting Things Done”

```

### 3. Steps to reproduce

```text
remark Kira_remark-lint-write-good__blockquotes_and_qoutes--non-valid-example.md --use remark-lint-write-good --no-stdout
```

### 4. Expected behavior

No warnings.

### 5. Actual behavior

```node
Kira_remark-lint-write-good__blockquotes_and_qoutes--non-valid-example.md
   3:5-3:14  warning  "therefore" is wordy or unneeded              write-good                       remark-lint
  3:31-3:47  warning  "be circumscribed" may be passive voice       write-good                       remark-lint
  7:10-7:16  warning  "Things" can weaken meaning                   write-good                       remark-lint

‼ 3 warnings
```

#### 6.1. Blockquotes

In example I use citation [**from “Hamlet”**](http://www.shakespeare-online.com/plays/hamlet_1_3.html). Shakespeare don't use write-good.

I can modify my own texts for write-good, but I can't modify text of another peoples.

#### 6.2. Quotes

“Getting Things Done” — is a [**David Allen time management method**](https://en.wikipedia.org/wiki/Getting_Things_Done). I can't remove or replace word “Things” from proper name.

Thanks.

**Comment by zerok** [comment]
Hi 🙂  Sorry for the late response 😞 https://github.com/zerok/remark-lint-write-good/commit/dff2c1ffaa29ce0da0b4a4ebd0e234066f86eab4 now contains at least a fix for your first issue. I'm not sure if I can do anything about ignoring quoted text, though. IMHO this is something that should be done in write-good.

**Comment by Kristinita** [comment]
@zerok , `type: question`

Can you publish your changes to npm?

Thanks.

**Comment by zerok** [comment]
@Kristinita will try to do that tonight 🙂

**Comment by zerok** [comment]
Done. It's now available as v1.1.0 :)
